services/mobile-client-integrations.adoc

```diff
@@ -0,0 +1,64 @@
+= Integrating Mobile Clients with Services in OpenShift
+
+Mobile Clients in OpenShift may be integrated with particular services provisioned in the namespace to allow communication between them. 
+The result of an integration produces a JSON configuration file (`mobile-services.json`) for your application which may be consumed by the Aerogear SDKs, allowing it to access the service (or services) it needs.
+
+== Create a Mobile Client
+A Mobile Client is a representation of your mobile application (Android, iOS, Cordova) in OpenShift. See link:./mobile-clients-in-openshift.adoc[this document] to learn more about Mobile Clients in OpenShift and how to create them. 
+
+== Integrating your Mobile Client with Services
+
+Once you have created your Mobile App Representation (MAR), it is possible to
```

Comment:
I REALLY dislike the term MAR!!!

Just because we are currently using it, is not a good enough reason to keep using it. "Mobile Client" would make a lot more sense to a lot more people.